File: matlab/Document/FibonacciSequence.m
disp("Fibonacci Sequence v1.0")
disp("Copyright (c) 2020 miniprime1")
disp(" ")
n = input('Enter Range: ');
disp(" ")
a = 0;
b = 1;
while a < n
    fprintf("%d, ", a)
    t = a;
    a = b;
    b = t + b;
end
disp(" ")

File: py/fibonacci_sequence.py
print("Fibonacci Sequence v1.0")
print("Copyright (c) 2020 miniprime1\n")
n = float(input("Enter limit: "))
print("")
a, b = 0, 1
while a < n:
    print(a, end=', ')
    a, b = b, a+b
print("")
input("\nPress enter key to exit")